core/shared/src/main/scala/io/udash/properties/single/Property.scala

```diff
@@ -253,5 +253,35 @@ trait Property[A] extends ReadableProperty[A] {
     */
   def transformToSeq[B : PropertyCreator](transformer: A => Seq[B], revert: Seq[B] => A): SeqProperty[B, Property[B]] =
     new SeqPropertyFromSingleValue(this, transformer, revert)
+
+  /**
+    * Bidirectionally synchronizes Property[B] with `this`.
```

Comment:
We should notice that it puts transformed value from `this` to `p` on initialization.